projects/go-lib/src/lib/go-config.service.ts

```diff
@@ -1,89 +1,62 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
-import { GoConfigInterface } from './go-config.model';
-
-interface RGB {
-  r: number;
-  g: number;
-  b: number;
-}
+import {
+  BrandingMode,
+  GoConfigInterface,
+  LogoConfig,
+  ThemeColors
+} from './go-config.model';
 
 @Injectable()
 export class GoConfigService {
-  config: BehaviorSubject<GoConfigInterface> = new BehaviorSubject<GoConfigInterface> ({
-    brandColor: '#65B360',
-    headerBrandingEnabled: false
-  });
+  private config: GoConfigInterface = {
+    brandColor: ThemeColors.brand,
+    brandingMode: BrandingMode.company,
+    logoConfig: {
+      logo: '',
+      logoLink: ''
+    }
+  };
 
   /**
-   * Use this method to specific a brand color to be used in all places where branding is applied
-   * @param color The color for the brand
+   * Configuration subject, you can subscribe to this to recieve change
+   * updates when the global config changes.
    */
-  public setBrandColor(color: string): void {
-    // we have to copy the config here or it won't regester a change in components
-    const config: GoConfigInterface = Object.assign({}, this.config.getValue());
-    config.brandColor = color;
-    this.config.next(config);
-  }
+  config$: BehaviorSubject<GoConfigInterface> = new BehaviorSubject<GoConfigInterface> (this.config);
```

Comment:
So I know this is pretty standard to add a $ in angular to designate a variable that is an observable, but we haven't been following that practice anywhere else in goponents. This seems like a breaking change that could be avoided here by setting the private config to a variable called `_config` and keeping this as just config. I know we can't avoid all the breaking changes in this PR and some of them I think are necessary, but it still seems like we should avoid the ones we can. Thoughts?

Reply:
I suppose that makes sense. I was initially just trying to adopt the naming convention and figured that if we were already making a few breaking changes in this version, what's one more... but you're probably right, we should limit it if we can. I've changed things to match your suggestion here.